rails/changelog.md

```diff
@@ -1,3 +1,13 @@
+## March 5th, 2018
+### New Components
```

Comment:
This will become available on the PR build after this merges: https://github.com/cerner/terra-kaiju-plugin/pull/18